pom.xml

```diff
@@ -336,12 +335,10 @@
     </profile>
     
     <!-- using github.com/google/error-prone-javac is required when running on JDK 8 -->
+    <!-- see instructions at https://errorprone.info/docs/installation -->
     <profile>
-      <id>errorprone8</id>
+      <id>errorprone-jdk8-fix</id>
```

Comment:
Nope, please don't rename the profile, documentation relates to it (and it's a fix now, but once fixed, there is no need to call it that way).

Reply:
Thanks.